docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,162 @@
+
+To solve this problem, a third microcontroller can be used as the new central or as more widely known a dongle. 
+The dongle will act as a central device and communicate with the keyboard halves which will both act as peripherals.
+Dongles are usually connected to the host device using USB. This way the central half of the keyboard can be powered by the host device, 
+and the peripheral halves can be powered by their own batteries. This setup allows both halves of the keyboard to have a longer battery life.
+
+{/* Here is an example diagram of a split keyboard with a dongle:
+
+```mermaid
+graph TD;
+    A[Computer/Host Device] <-- USB/BLE --> DONGLE[Dongle/Central];
+    DONGLE -. BLE .-> LEFT[Left Half/Peripheral];
+    DONGLE -. BLE .-> RIGHT[Right Half/Peripheral];
+```
+
+Dotted lines represent BLE connections.
+Solid lines represent USB connections. */}
+
+:::info
+For more information on how to set up a two split keyboard, refer to [New Keyboard Shield](../development/new-shield.mdx) under Guides.
+After setting up the split keyboard, you can follow the steps below to add a dongle to the setup.
+:::
+
+## Benefits/Disadvantages of using a Dongle/Split setup
+
+Benefits:
+
+- Longer battery life for peripheral halves of the keyboard.
+- Dongle can be powered by the host device and no long wires are needed.
+- Easier to connect to the host device.
+- Only the central side needs to reflashed for keymap updates
+
+Disadvantages:
+
+- More complex setup.
+- More components needed.
+- Added latency due to the extra hop between the host device and the keyboard halves.
+- Each half of the keyboard needs its own firmware (ex: left, right, and dongle).
+
+:::warning[Reset/Bootloader behavior]
+When using a combo to trigger reset/bootloader, only the central side will be affected. To avoid this, you can avoid using the combo and instead use a physical reset button on the central half.
+That will allow you to locally reset the central/peripheral side without affecting other halves. 
+Further explanation can be found under [Behavior Reset#split-keyboards](../behaviors/reset.md#split-keyboards)
+:::
+
+## Building a Split Keyboard shield with a Dongle
+
+:::note
+Because we will be changing the roles of the keyboard halves, we will also need to flash the reset firmware to the keyboard halves.
+Information about the reset firmware can be found under Troubleshooting [Reset firmware](../troubleshooting/connection-issues.mdx#acquiring-a-reset-uf2)
+:::
+
+### Kconfig.shield
+
+First we will introduce a third split to our keyboard configuration. This will be used as the dongle.
+
+```kconfig
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD_LEFT
+    def_bool $(shields_list_contains,my_board_left)
+
+config SHIELD_MY_BOARD_RIGHT
+    def_bool $(shields_list_contains,my_board_right)
+```
```

Comment:
I think you could emphasize that this is an edit and addition to the existing file a bit more.

Reply:
Added clarification above